packages/core/src/App/Components/Elements/LiveChat/live-chat.jsx

```diff
@@ -65,40 +65,42 @@ const LiveChat = ({ is_mobile_drawer, has_cookie_account }) => {
-                    is_logged_in: !!is_logged_in,
-                    loginid: loginid ?? '',
-                    landing_company_shortcode: landing_company_shortcode ?? '',
-                    currency: currency ?? '',
-                    residence: residence ?? '',
-                    email: email ?? '',
-                    utm_source: utm_source ?? '',
-                    utm_medium: utm_medium ?? '',
-                    utm_campaign: utm_campaign ?? '',
-                };
-                window.LiveChatWidget.call('set_session_variables', session_variables);
-
-                if (is_logged_in) {
-                    // client logged in
-                    // prepfill name and email
-                    window.LiveChatWidget.call('set_customer_email', session_variables.email);
-                    window.LiveChatWidget.call('set_customer_name', `${client_first_name} ${client_last_name}`);
+                if (window.LiveChatWidget) {
+                    const session_variables = {
+                        is_logged_in: !!is_logged_in,
+                        loginid: loginid ?? '',
+                        landing_company_shortcode: landing_company_shortcode ?? '',
+                        currency: currency ?? '',
+                        residence: residence ?? '',
+                        email: email ?? '',
+                        utm_source: utm_source ?? '',
+                        utm_medium: utm_medium ?? '',
+                        utm_campaign: utm_campaign ?? '',
+                    };
+                    window.LiveChatWidget.call('set_session_variables', session_variables);
 
-                    // prefill name and email fields after chat has ended
-                    window.LC_API.on_chat_ended = () => {
+                    if (is_logged_in) {
+                        // client logged in
+                        // prepfill name and email
                         window.LiveChatWidget.call('set_customer_email', session_variables.email);
                         window.LiveChatWidget.call('set_customer_name', `${client_first_name} ${client_last_name}`);
-                    };
-                } else {
-                    // client not logged in
-                    // clear name and email fields
-                    window.LiveChatWidget.call('set_customer_email', ' ');
-                    window.LiveChatWidget.call('set_customer_name', ' ');
-                    // clear name and email fields after chat has ended
-                    window.LC_API.on_chat_ended = () => {
+
+                        // prefill name and email fields after chat has ended
+                        window.LC_API.on_chat_ended = () => {
+                            window.LiveChatWidget.call('set_customer_email', session_variables.email);
+                            window.LiveChatWidget.call('set_customer_name', `${client_first_name} ${client_last_name}`);
+                        };
+                    } else {
```

Comment:
there's another approach for these if and else statements
in the if statement  you can add a return 
and remove the else statement because in case of going through that if it will return and also after the if you can have a return statement too.
It's not a bug and you can ignore this approach, it just a suggestion. ( it is a little better in performance )
https://stackoverflow.com/questions/1820839/using-return-instead-of-else-in-javascript